NEWS.rst

```diff
@@ -45,6 +45,7 @@ Bug fixes
   duplicates another *inactive* association.
 * #25704 Added max-width to table columns.
 * #25696: Added remove button for dates.
+* #25694: Table layout is now fixed.
```

Comment:
Erm, could you perhaps elaborate on this a bit? I assume something was wrong in order for it to need a fix?